design/src/high-level/game-thesis.md

```diff
@@ -0,0 +1,176 @@
+# *Emergence* within the Factory Builder Genre
+
+Following the [genre analysis](genre-analysis.md), let's examine how *Emergence* fits within the factory builder genre.
+
+## Game Thesis
+
+With a biologically inspired setting and game mechanics,
+*Emergence* attempts to fill a unique niche in a very succesful genre that is low on genuine innovation.
+
+*Factorio*, a smash hit indie game, succeded because of its:
+
+- relatively intuitive, easy to control UX
+- satisfying, simple audio
+- clear graphics
+- rich and interesting contraption mechanics
+- well-paced progression
+- thriving modding community
+- impressive performance
+- intriguing fantasy: build a factory on an alien world and advance in technology
+
+However, competitors in the space have often failed to catch audience's attention in the same way.
+Getting the details right (particularly around UX, contraption mechanics, progression and performance) is remarkably hard.
+Just as importantly, these competitors are by-and-large unimaginative, both in theme and mechanics.
+Belts, inserters and machines in a generic sci-fi environment are all well-and-good,
+but fail to capture the imagination of new audiences, or stand out from the crowd.
+
+By taking the rich and fun core mechanics of factory builders,
+adapting them to a unique and compelling theme,
+and adding a unique mechanical focus on resilience,
+*Emergence* looks to appeal to a loyal audience ravenous for something meaningfully new.
+
+## Mechanical Translation
+
+### Core Mechanics
+
+- **Resource patches**
+  - Resources are continuous, rather than discrete.
```

Comment:
But are all resources continuous? Aren't some things simply discrete? 

Related: how is food output from plants modelled? Does a plant produce integer units of food, which ants and so on and can take varying units of? Or does plant produce continuous units of food? If so, what does that buy us?

Reply:
```suggestion
  - Environmental resources are continuous, rather than discrete.
```